### PR TITLE
[close #217] Add BR unit test coverage statistics

### LINKED ISSUE #217: BR: Add btikv-br unit coverage statistics.
## Feature Request

**Is your feature request related to a problem? Please describe:**

Add btikv-br unit coverage statistics.

**Describe the feature you'd like:**

Add btikv-br unit coverage statistics.
**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by haojinming**
Signed-off-by: haojinming <jinming.hao@pingcap.com>

### What problem does this PR solve?

Issue Number: close #217 

Problem Description: 

### What is changed and how does it work?

Add code coverage statistics for tikv-br

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: br/Makefile [hunk 1]
```diff
@@ -57,6 +57,7 @@ test: tools/bin/gocov tools/bin/gocov-xml
 	$(GO) test -p $(TEST_PARALLEL) -race -ldflags '$(LDFLAGS)' -tags leak $$($(PACKAGES)) -coverprofile=$(COVERAGE_DIR)/coverage.raw || ( make failpoint/disable && exit 1 )
 	tools/bin/gocov convert $(COVERAGE_DIR)/coverage.raw | tools/bin/gocov-xml > $(COVERAGE_DIR)/coverage.xml
 	make failpoint/disable
+	bash <(curl -s https://codecov.io/bash) -F cdc -f $(COVERAGE_DIR)/coverage.raw -t $(TIKV_MIGRATION_CODECOV_TOKEN)
```
Comment (pingyu) [inline br/Makefile:60]: Suggest to add another Makefile target for CI. Otherwise `make test` locally (used by daily development) will meet some errors.

**Review by pingyu — COMMENTED** [review]
Rest LGTM.

File: br/pkg/backup/check.go [hunk 2]
```diff
@@ -12,14 +12,16 @@ import (
 )
 
 // checkDupFiles checks if there are any files are duplicated.
-func checkDupFiles(rangeTree *rtree.RangeTree) {
+func checkDupFiles(rangeTree *rtree.RangeTree) bool {
```
[the remark this answers is not in the document]
Comment (haojinming) [reply br/pkg/backup/check.go:15]: yes, only log in printed. If duplicated file occurs, it has no side effect but the performance when restore. I'm not sure why deplicated files occurs. Maybe grpc send/recv twice when network problem occurs like flash breakdown, need more investigation.

**Review by pingyu — APPROVED** [review]
LGTM~

**Comment by pingyu** [comment]
@zeminzhou PTAL~

**Review by zeminzhou — APPROVED** [review]
LGTM~

**Pull request merged**